CMSgov/mint-app
# [MINT-2449, MINT-2997] Update Read View Tooltips

# MINT-2449, MINT-2997

## Description

* Added  question tooltips for the following:
  * Operations Evaluation and Learning
  * Payments

## How to test this change

- In any model plan, fill in the following:
    - Answer `Are you planning to use IDDOC support?` in page 1 of Operations, evaluation, and learning 
    - Select both `Claims-based payments` and `Non-claims-based payments` in Page 1 of Payments
    - Select `Reduction to beneficiary cost-sharing` for the question `Select which claims-based payments will you pay.` (page 2 of Payments)
    - Answer `Will any additional data need to be collected for claims?` in Page 3 of Payments
    - Answer `Will you waive beneficiary cost-sharing (coinsurance and/or deductible) for any services in the model?` in page 4 of Payments
    - Answer `Would the waiver only apply for part of the payment?` in page 4 of Payments
    - Answer `Are you planning to use the Innovation Payment Contractor?` in page 5 of Payments
- Save and go to Read view
- Ensure all tooltips in AC are shown in every read-view and have the correct tooltip
 

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
